view/admin/actions/index.js

```diff
@@ -152,7 +151,6 @@ function requestWithOutPayload(verb, url) {
   return new Promise((resolve, reject) => {
     superagent(verb, API_URL + url)
     .set('Content-Type', 'application/json')
-    .set('Authorization', u.getCookie('Authorization'))
```

Comment:
Yes I guess because its general function for cookies not just for authorization cookies.